styleguide/derek/global-components/tracking/tracking.js

```diff
@@ -28,6 +28,17 @@ Zoolander.Tracking = (function Tracking($) {
       });
     });
 
+    $('.track-ceilingCall').on('click', () => {
```

Comment:
So looking at this a little bit closer, it looks like we're doing double tracking now.  We should remove the `ga.event` that is happening above, and then just modify the `rs.call_click` in GTM to pass in the `transport` value and custom dimensions. See `GA - Generic Event` inside GTM for reference.